app/components/subnav_component/nav_item_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<li class="app-subnav__list-item<%= current_page?(path, check_parameters: true) ? " app-subnav__list-item--selected" : "" %>">
```

Comment:
This was preventing the tab from highlighting if there were any query params, for example /admin/users?page=2 would stop highlighting the users tab because the params didn't match the URL on the button.